serving/docs/lmi_new/user_guides/tnx_user_guide.md

```diff
@@ -0,0 +1,115 @@
+# Transformers-NeuronX Engine in LMI
+
+## Model Artifacts Structure
+
+LMI Transformers-NeuronX expect the model to be standard HuggingFace format for runtime compilation. For loading of compiled models, both Optimum compiled models and split-models with a separate `neff` cache (compiled models must be compiled with the same Neuron compiler version and model settings).
+The source of the model could be:
+
+- model_id string from huggingface
+- s3 url that store the artifact that follows the HuggingFace model repo structure, the Optimum compiled model repo structure, or a split model with a second url for the `neff` cache.
+- A local path that contains everything in a folder follows HuggingFace model repo structure, the Optimum compiled model repo structure, or a split model with a second directory for the `neff` cache.
+
+### Standard HuggingFace model artifacts
```

Comment:
please move model artifacts description to that description doc @siddvenk created

Reply:
Moved to the new location